roles/deprecated_api/README.md

```diff
@@ -25,26 +18,33 @@ The role requires the following:
 - Ensure the availability of the K8s [APIRequestCount](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/assets/kube-apiserver/apiserver.openshift.io_apirequestcount.yaml) object. For OpenShift, APIRequestCount is included in the standard distribution from OCP 4.9 onwards. For other distributions, deploy the required CRD and CR before using the role.
 - Python version 3.6 or higher is needed to parse the APIs.
 
-
 # Parameters
 
 | Parameter               | Choices/Defaults      | Comments                                             |
 |-------------------------|-----------------------|------------------------------------------------------|
 | `kubeconfig`            | `~/.kube/config.json` | Path to an existing Kubernetes config file to be used by k8s_info dependency. If not provided, the OpenShift client will attempt to load the default configuration file from `~/.kube/config.json`. Can also be specified via `K8S_AUTH_KUBECONFIG` environment variable.                                                                                                |
-| `deprecated_namespaces` | `undefined`           | List of namespaces the role will check for deprecated APIs. Typically, it should include the namespaces where you deploy the workload. If left undefined, the role will check all namespaces, excluding those starting with the `openshift` and `kube-` prefixes.                                                |
+| `deprecated_namespaces` | `undefined`           | Optional. List of namespaces the role will check for deprecated APIs. Typically, it should include the namespaces where you deploy the workload. If left undefined, the role will check all namespaces, excluding those starting with the `openshift` and `kube-` prefixes.                                                |
 | `ocp_version`           | `"4.9"/"4.10"/etc`    | OpenShift version currently running on your cluster. |
-| `job_logs`              | `undefined`           | Directory to store the output JUnit file containing the 'Workload Compatibility with OCP Versions' testsuite.                                                              |
+| `deprecated_api_logs`   | `/tmp`                | Optional. Directory to store the output JUnit file containing the 'Workload Compatibility with OCP Versions' testsuite.                                                              |
```

Comment:
> should it be `deprecated_api_logs.path` instead?

Thank you, @thekad. Let me fix the docs.

> also is it your plan to use a dictionary?

For the moment, yes. `deprecated_api_logs` is a local copy of `job_logs` which we use everywhere as `job_logs.path`.
It's true that other parts of the `job_logs` dictionary are never used, and it would be nice to change the variable from the dictionary to a plain path one day.

Reply:
Hi @thekad, I fixed the docs, could you please have a look one more time?